content/components/dialog.mdx

```diff
@@ -313,6 +321,36 @@ Side sheets take the entire height of the viewport, and should be used sparingly
   configuration panels.
 </Caption>
 
+Side sheets are still considered as dialogs, which means that interacting with the content in the background is not possible.
+This needs to be indicated by a [backdrop](#backdrop) and should always cover the global navigation.
+This avoids that multiple side sheets can appear simultaneously.
+
+<DoDontContainer>
+  <Do>
+    <img width="464" alt="" src="https://github.com/primer/design/assets/980622/6501c520-70da-49a2-8b29-a9f028c6a6e3" />
+    <Caption>Side sheets appear on top of the page (including the navigation).</Caption>
+  </Do>
+  <Dont>
+    <img width="464" alt="" src="https://github.com/primer/design/assets/980622/f05aa559-a866-47cf-84a2-f4897f1da08c" />
+    <Caption>Avoid side sheets that keep ineractions with the rest of the page enabled.</Caption>
+  </Dont>
+</DoDontContainer>
+
+Right side sheets serve multiple purposes, including facilitating global actions and providing quick previews in full-width pages.
+However, it is important to exercise caution when implementing side sheets in pages that are not full-width. The objective is to minimize the distance users need to travel with their mouse cursor to access the side sheet, especially on wide screens.
+This consideration is essential for maximizing productivity.
+
+<DoDontContainer>
+  <Do>
+    <img width="464" alt="" src="https://github.com/primer/design/assets/980622/84585f5e-fad5-451c-8c53-87972f69030d" />
+    <Caption>Use side sheets to provide a quick preview in full-width pages.</Caption>
+  </Do>
+  <Dont>
+    <img width="464" alt="" src="https://github.com/primer/design/assets/980622/df59604f-c459-4aff-8db7-3e23000c91ac" />
+    <Caption>Unless it's a global action avoid side sheets in pages that aren't full-width.</Caption>
```

Comment:
I was a little confused by "pages that aren't full-width". I had to stare at the "do" and "don't" examples to figure out what you meant.

```suggestion
    <Caption>Unless it's a global action, avoid side sheets in pages where content does not stretch the full width of the page.</Caption>
```